numpy/f2py/_src_pyf.py

```diff
@@ -143,7 +145,7 @@ def listrepl(mobj):
             lnames[name] = thelist
         return "<%s>" % name
 
-    substr = list_re.sub(listrepl, substr) # convert all lists to named templates
+    substr = list_re.sub(listrepl, substr)  # convert all lists to named templates
                                            # newnames are constructed as needed
```

Comment:
Might want to line this up. Maybe better, put the comments on the preceding line. I hate trailing comments :)